src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -110,6 +111,7 @@ type SlotConfig =
         ApplyIPSecurityRestrictionsToScm: bool
         EnablePublicNetworkAccess: bool option
         AlwaysOn: bool option
+        NetFrameworkVersion: string option
```

Comment:
Might be better to store a `Runtime` here to match what's in the WebApp - this will also keep us more flexible if we need to make other shanges in future